### ISSUE TITLE
How to fix a problem with single quotes?

### ISSUE THREAD

**Issue by danilvalov** [josdejong/mathjs#816]
I have the following example:
```
var math = require('mathjs');
var str = "test('arg')";
console.log(math.parse(str));
```

And when I try to run it I have the following error:
```
SyntaxError: Value expected (char 6)
    at createSyntaxError (/test/node_modules/mathjs/lib/expression/parse.js:1524:17)
    at parseEnd (/test/node_modules/mathjs/lib/expression/parse.js:1492:13)
    at parseParentheses (/test/node_modules/mathjs/lib/expression/parse.js:1479:12)
    at parseNumber (/test/node_modules/mathjs/lib/expression/parse.js:1449:12)
    at parseObject (/test/node_modules/mathjs/lib/expression/parse.js:1430:12)
    at parseMatrix (/test/node_modules/mathjs/lib/expression/parse.js:1357:12)
    at parseString (/test/node_modules/mathjs/lib/expression/parse.js:1259:12)
    at parseSymbol (/test/node_modules/mathjs/lib/expression/parse.js:1143:12)
    at parseCustomNodes (/test/node_modules/mathjs/lib/expression/parse.js:1120:12)
    at parseLeftHandOperators (/test/node_modules/mathjs/lib/expression/parse.js:1034:12)
```

If I change single quotes in the string to double quotes it works correctly.

**Comment by josdejong** [comment]
The parser currently only supports double quotes. I myself regularly enter single quotes too, as I'm used to that in JavaScript. Makes sense to me to add support for single quotes too. Let's put this as a feature request.

**Comment by josdejong** [comment]
And thanks for your suggestion btw :)

**Comment by jean-emmanuel** [comment]
PR #1202 :)

**Comment by josdejong** [comment]
Fixed via #1202, thanks @jean-emmanuel!
